CHANGES.rst

```diff
@@ -49,6 +49,9 @@ Specviz
 Other Changes and Additions
 ---------------------------
 
+- ``jdaviz`` no longer uses ``spectral-cube``. Cubeviz now loads data cube
```

Comment:
This is not quite true yet, but hopefully will be true by the time this PR is complete.